06.bowling_object/frame.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class Frame
+  attr_reader :frame_number
+
+  def initialize(frame_number)
+    @frame_number = frame_number
+    @shot_scores = []
+    @bonuses = []
```

Comment:
```suggestion
    @bonus_scores = []
```

とした方が他の命名と揃ってわかりやすそうです。（bonusの複数形ってあまり馴染みがないですし）

Reply:
他の変数との親和性？で可読性が上がるっていうのは考えたことなかったですが、たしかにその通りだと思いました。
そのように修正しました